Tenets

```diff
@@ -0,0 +1,9 @@
+This document outlines the OpenSearch Security teams tenets. 
```

Comment:
There were a couple of tenets that we were considering, did you want to add them into this doc?

Reply:
Removed the tenets file since I don't think we need a separate file for it. I do think keeping the tenets in a "How We Work" section in the DEVELOPER GUIDE is a better home for that content.

Feel free to review and suggest updates :D